src/main.ts

```diff
@@ -36,7 +36,7 @@ export default async function (api: (token: string) => API): Promise<void> {
   const filePath = `Formula/${formulaName}.rb`
   const tagName = (process.env.GITHUB_REF as string).replace('refs/tags/', '')
   const tagSha = process.env.GITHUB_SHA as string
-  const version = tagName.replace(/^v(\d)/, '$1')
+  const version = getInput('formula-version') || tagName.replace(/^v(\d)/, '$1')
```

Comment:
Hi, how about naming this `tag-name` and accepting a git tag in either `v1.2.3` or `refs/tags/v1.2.3` format as substitute for the GITHUB_REF input? I'm thinking this approach is more flexible since it also allows updating git-based Homebrew formulae.

We could also support an optional `tag-sha` input and, if not given, default to resolving the `tag-name` to a commit SHA via the GitHub API.

Reply:
Renamed `formula-version` to `tag-name`.

Reply:
Not really part of this PR, but could we make the leading `v` in tag name optional? So change `/^v(\d)/` into `/^v?(\d)/`. Because e.g. the project I'd be using this action doesn't use the leading `v` in tag names, it just has the version number.